lib/MusicBrainz/Server/WebService/Validator.pm

```diff
@@ -81,8 +81,6 @@ sub load_type_and_status
         lc('sa-' . $_->name)=> $_->id,
         lc('va-' . $_->name)=> $_->id,
     } @statuses;
-
-    $types{'nat'} = $types{'non-album tracks'};
```

Comment:
The first is for historic edits, so it's probably never going to run anyway and might be good to have it for historical documentation? I mean, we could also choose to just get rid of all non-display code for historical edits too, that'd probably be sensible, but otherwise I wouldn't just remove random bits of it :) 

The second seems sensible, we don't seem to use NONALBUMTRACKS_NAME nor RELEASE_ATTR_NONALBUMTRACKS anywhere at all :) Will remove those.